be/src/env/env_stream_pipe.h

```diff
@@ -10,7 +10,8 @@ class StreamLoadPipe;
 // Since the StreamPipe does not have the standard
 // SequentialFile interface. The StreamPipeSequentialFile
 // is used to wrap the memory to provide the sequential API.
-class StreamPipeSequentialFile : public SequentialFile {
```

Comment:
StreamPipeSequentialFile -> PipeInputStream?

Reply:
Good suggestion! How do you think about `StreaLoadPipeInputStream`?